enclave/core/sgx/td.c

```diff
@@ -168,6 +168,23 @@ oe_sgx_td_t* oe_sgx_get_td()
     return td;
 }
 
+/*
+**==============================================================================
+**
+** oe_sgx_set_td_exception_handler_stack()
+**
+**     Internal API that allows an enclave to setup stack area for
+**     exception handlers to use.
+**
+**==============================================================================
+*/
+void oe_sgx_set_td_exception_handler_stack(void* stack, uint64_t size)
+{
+    oe_sgx_td_t* td = oe_sgx_get_td();
```

Comment:
It would be good to ensure that stack+size is 16 byte aligned even though oe_enter aligns the stack.
It makes the expectation clear.